pkg/types/date.go

```diff
@@ -930,7 +942,11 @@ func (PatternDateTimeParser) SetNodeValue(node graph.Node, value interface{}) er
 		if _, ok := node.GetProperty(GoTimeFormatTerm); ok {
 			ls.SetRawNodeValue(node, v.Format(ls.AsPropertyValue(node.GetProperty(GoTimeFormatTerm)).AsString()))
 		} else {
-			ls.SetRawNodeValue(node, v.Format(ls.AsPropertyValue(node.GetProperty(MomentTimeFormatTerm)).AsString()))
+			goment, err := ToGomentTime(v)
```

Comment:
Rewrite these so that, at the beginning of PatternXXX:
if s, ok:=node.GetProperty(GoTimeFormatTerm); ok{
    formatter:=goFormat(ls.AsPropertyValue(s).AsString())
}
if s, ok:=node.GetProperty(GomentTimeFormatTerm); ok{
    formatter:=gomentFormat(ls.AsPropertyValue(s).AsString())
}

if formater=nil {
  return nil,error (missing formatting annotations)
}